### DISCUSSION TITLE
What extensions are not currently supported?

### DISCUSSION THREAD

**Discussion by cykabass** [ImageMagick/ImageMagick#3268] [Help]
Hi could someone provide a list of file extensions ImageMagick currently does not support?

Thanks.

**Comment by snibgo** [comment]
An infinite number of file extensions are not supported.

For finite list of file extensions generally supported, see https://www.imagemagick.org/script/formats.php#supported .

For the finite number of file extensions that are supported by your installation:
```
magick -list format
```
